### DISCUSSION TITLE
Cannot log in from google

### DISCUSSION THREAD

**Discussion by cheezypotatoes** [ultrafunkamsterdam/undetected-chromedriver#1231] [Q&A]
Last night, it was fine and dandy. 
![image](https://user-images.githubusercontent.com/132412990/235811995-11837080-cd51-4b18-8622-bb6023c76593.png)

import time
import undetected_chromedriver as uc
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.chrome.options import Options
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC

if __name__ == '__main__':

    my_email = ""
    my_password = ""

    driver = uc.Chrome()

    driver.get("https://accounts.google.com/v3/signin/identifier?dsh=S-1710339674%3A1683071284006015&continue=https%3A%2F%2Fwww.youtube.com%2Fsignin%3Faction_handle_signin%3Dtrue%26app%3Ddesktop%26hl%3Den%26next%3Dhttps%253A%252F%252Fwww.youtube.com%252F%253FthemeRefresh%253D1&ec=65620&hl=en&ifkv=Af_xneGypjuIjTcBGy2crr9h3KC907SJQFCC4pMLRDVVye_Jq4YC2e_73ycwqhU55OyoWUFyw9g7&passive=true&service=youtube&uilel=3&flowName=GlifWebSignIn&flowEntry=ServiceLogin")

    wait = WebDriverWait(driver, 50)

    email_input = wait.until(EC.presence_of_element_located((By.ID, "identifierId")))
    email_input.send_keys(my_email)

    next_button = wait.until(EC.presence_of_element_located((By.XPATH, '//*[@id="identifierNext"]/div/button')))
    next_button.click()

    password_input = wait.until(EC.presence_of_element_located((By.NAME, "Passwd")))
    password_input.send_keys(my_password)

    time.sleep(30)

**Comment by devlamgi** [comment]
You can use undetected from seleniumbase instead. I've tried this patch and logged successfully

**Comment by TheGreatSimo** [comment]
@devlamgi how, can you please explain how

**Reply by devlamgi** [reply]
```
from seleniumbase import undetected as uc
```
just that
they have patch this PR I think https://github.com/ultrafunkamsterdam/undetected-chromedriver/pull/1010
